omnibus/omnibus-test.sh

```diff
@@ -120,16 +120,14 @@ export CHEF_LICENSE=accept-no-persist
 
 cd "$chef_gem"
 
-# only add -E if not on centos 6
-sudo_path="$(command -v sudo)"
-# cspell:disable-next-line
-rhel_sudo="/opt/rh/devtoolset-7/root/usr/bin/sudo"
-sudo_args=""
-if [[ "$sudo_path" != "$rhel_sudo" ]]; then
-  echo "HERE"
+# # only add -E if not on centos 6
+# sudo_path="$(command -v sudo)"
+# # cspell:disable-next-line
+# rhel_sudo="/opt/rh/devtoolset-7/root/usr/bin/sudo"
+# if [[ "$sudo_path" != "$rhel_sudo" ]]; then
   sudo -E bundle install --jobs=3 --retry=3
   sudo -E bundle exec rspec --profile -f progress
-else
-  sudo bundle install --jobs=3 --retry=3
-  sudo bundle exec rspec --profile -f progress
-fi
+# else
+#   sudo bundle install --jobs=3 --retry=3
+#   sudo bundle exec rspec --profile -f progress
+# fi
```

Comment:
You said you're reverting a PR - can you link to it in the PR description?

Also, does this FIX C6? If so, lets just delete this code, not comment it out.

If not, why are we dropping C6 code?

Reply:
Added the PR to descripton.

The code was to fix C6 running build/test inside a container, I removed it for now and will re-add when we are ready to use the container.